### DISCUSSION TITLE
Unit testing

### DISCUSSION THREAD

**Discussion by nandanvasudevan** [cnjinhao/nana#667] [Ideas]
Hi I don't see any unit tests.
Can we use Catch2 or any other if you have anything else already in mind?

**Comment by qPCR4vir** [comment]
Hi, the repo for testing is nana_demo. I have no plan for Unit testing, but if you have a PR, I will open there a new branch and will try to setup again a CI including your Tests. We have there 50 functional test. Testing GUI is Hard. Thank

**Reply by qPCR4vir** [reply]
https://github.com/qPCR4vir/nana-demo
